packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -102,24 +103,15 @@ const TeamPromptMeeting = (props: Props) => {
     `,
     meetingRef
   )
-
+  const {phases} = meeting
   const maybeTabletPlus = useBreakpoint(Breakpoint.FUZZY_TABLET)
-  const teamMembers = useMemo(() => {
-    return meeting.phases
-      .flatMap((phase) => phase.stages)
-      .flatMap((stage) => stage.teamMember)
-      .map((teamMember) => {
-        if (!teamMember) {
-          return null
-        }
 
-        return {
-          ...teamMember,
-          key: teamMember.id
-        }
-      })
-      .filter(isNotNull)
-  }, [meeting.phases])
+  const phase = getPhaseByTypename(phases, 'TeamPromptResponsesPhase')
```

Comment:
@BartoszJarocki let me know if this gets you whatcha need. if so, go ahead & merge this PR & we'll start using this more in the future

Reply:
yup, looks good! thank you!